### ISSUE TITLE
koji-hub host setup

### ISSUE THREAD

**Issue by odra** [fedora-infra/mbbox#70]
Depends on #61

## Acceptance Criteria

* Setup a koji-hub host as part of a koji-builder setup;
* Add a host using the same service used by koji-hub;
* Should use the admin cert created by the koji-hub role;
* The builder should create a client certificate that uses the host hostname in its CN field;
* Use x86 arch for now;
* Bonus:  create an ansible module using koji as a package.

## Definition of Done

Koji host added and enabled as part of koji-builder deployment.

## Notes

The following commands can be used to accomplish this task:

```bash
# https://fedoraproject.org/wiki/Koji/ServerHowToProposed#Add_the_host_to_the_createrepo_channel
koji add-host koji-hub:8443 x86
koji add-host-to-channel koji-hub:8443 createrepo
```

It would be nice to have a ansible module that uses koji library to achieve the same results instead of running commands.

If none of those work we might need to create an "admin koji pod" that contains koji cli configured.
